db/seeds.rb

```diff
@@ -17,6 +10,14 @@
 
 puts "Employees table seeding done"
 
+puts "Admins table seeding..."
+    admin = Admin.new
+    admin.email = "admin@rera.com"
+    admin.password = 'zaq123'
+    admin.password_confirmation = 'zaq123'
+    admin.save!
```

Comment:
Consider using  `create!` method instead.
Also - do you really need to provide the `password_confirmation` during the object creation? 

```suggestion
    admin.create!(email: 'admin@rera.com', password: 'zaq123')
```